internal/kibana/kibana_test.go

```diff
@@ -160,82 +155,6 @@ var _ = Describe("Reconciling", func() {
 			})
 		})
 
-		Context("when updating kibana on an existing cluster", func() {
```

Comment:
why are we dropping this test?

Reply:
This test was dedicated to the removal of the old kibana shared config map which was requested above to remove as obsolete.